main.py

```diff
@@ -344,53 +374,95 @@ def __init__(self, *args):
 		self.inst = Xqemu()
 		self.settings = SettingsManager()
 		self.settings.load()
-		self.runButton.setText('Start')
-		self.pauseButton.setText('Pause')
-		self.screenshotButton.setText('Screenshot')
+		self.fileformats = [
+			{
+				'name': 'ISO9660 CD/DVD image file',
+				'offset': [
+					0x8001,
+					0x8801,
+					0x9001
+				],
+				'signature': [
+					'43 44 30 30 31'
```

Comment:
Why store it like this and not just 'CD001'?

Reply:
because it gets compared to bytes

Reply:
b'CD001'?